src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').reduce((stylesObject, style) => {
+    const cleanedStyle = style.trim().replace(/;$/, '');
```

Comment:
The regular expression `/;$/` is used to remove a semicolon at the end of the string. However, since the `split(';')` method is used on line 9, there should not be any trailing semicolons in the `style` variable. This line is redundant and can be removed.